views/apartments/show.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title><% @apartment.title %> </title>
+</head>
+<body>
+  <p>
+
+    <a href class="song.">
+      <p>
+        <%= "Address: " + @tenant.address %>
```

Comment:
address isn't a property of a tenant. I think you want to leverage `@apartment` that you defined in your `app.rb`